dvr_scan/scanner.py

```diff
@@ -367,13 +385,15 @@ def scan_motion(self):
                                 self.video_resolution)
                         for frame in buffered_frames:
                             if self.draw_timecode:
-                                self._stampText(frame, curr_pos.get_timecode(), 0)
+                                self._stampText(
+                                    frame, curr_pos.get_timecode(), 0)
                             video_writer.write(frame)
                         buffered_frames = []
 
             curr_pos.frame_num += 1
             num_frames_read += 1
             num_frames_processed += 1
+            self.frames_processed = num_frames_processed
```

Comment:
I added this property to the class so the GUI can access this to calculate the progress.